main.py

```diff
@@ -61,3 +64,22 @@ def run_bash_javascript(input_code):
         out, error = process.communicate()
         string_output = out.decode('utf-8')
         return json.loads(string_output)
+
+"""
+Add semicolons to the end of java lines
+"""
+def java_linter(output_code):
```

Comment:
should this be in main.py? Feel like it should be in one of the java folders?

Reply:
Yeah, I agree but I don't know where exactly makes sense

Reply:
Can this live in gast_to_code helpers? It seems like JavaScript should have this same problem.